src/Axes.ts

```diff
@@ -401,6 +401,60 @@ export default class Axes extends Component<AxesEvents> {
 		return this;
 	}
 
+	/**
+	 * Stop an animation in progress.
+	 * @ko 재생 중인 애니메이션을 정지한다.
+	 * @method eg.Axes#stop
+	 * @return {eg.Axes} An instance of a module itself <ko>모듈 자신의 인스턴스</ko>
+	 * @example
+	 * const axes = new eg.Axes({
+	 *   "x": {
+	 *      range: [0, 100]
+	 *   },
+	 * });
+	 *
+	 * axes.setTo({"x": 10}, 1000); // start animatation
+	 *
+	 * // after 500 ms
+	 * axes.stop(); // stop animation during movement.
+	 */
+	stop() {
```

Comment:
If Axes play one role, the stop name might be fine, but the stopAnimation name would be more accurate.